filebeat/input/log/config.go

```diff
@@ -105,7 +105,7 @@ var ValidScanOrder = map[string]struct{}{
 	ScanOrderDesc: {},
 }
 
-// ValidScanOrder of valid scan orders
+// ValidScanSort of valid scan sorts
```

Comment:
What do you think of:

```suggestion
// ValidScanSort is a map of valid scan sorts
```

Reply:
Thanks, I will do it right now.